R/boot_ipw.R

```diff
@@ -238,7 +229,7 @@ bootIPW_multicore <- function(X_rand,
   rep_weights <- survey::as.svrepdesign(svydesign, type = rep_type, replicates = num_boot)$repweights$weights
 
   k <- 1:num_boot
-  mu_hats_boot <- foreach::`%dopar%`(
+  mu_hats_boot <- mu_hats_boot <- foreach::`%dopar%`(
```

Comment:
@LukaszChrostowski  surely this is a mistake, right double assignment to the same variable?